client/src/types/statistic.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * This file contains the types for trap speed payloads
+ */
+
+import { Number, Record, Static } from 'runtypes';
+
+export const SpeedPayload = Record({
+  speed: Number,
+});
+export type SpeedPayload = Static<typeof SpeedPayload>;
```

Comment:
Both the `Runtype` and the`type` SpeedPayload are named the same. I believe for RunTypes we use an `RT` suffix and for type we use `T` by convention, so `SpeedPayloadRT` on line 7 and `SpeedPayloadT` on line 10. This isn't a bug, everything is still working nicely - but it beats me as to how it's working 😂😂. 

We can merge this in though and maybe we can refactor this in a later PR :)

Reply:
So actually I talked to Angus T a while back and it turns out this works because one is a variable and the other is a type declaration (different implementations in TS). I think this is supposed to be the best practise way of doing things but I didn't know this before hence `R` and `RT`

Reply:
ohhh interesting. Just out of curiosity though, taking this PR as an example, `SpeedPayload` is imported in `StatisticRow.tsx` and it's used as a type in `useCallback` but as a RunType in `useChannelShaped` argument, so is JavaScript smart enough to replace the right `Speedpayload` in the right place? Plus how does it know when importing to import the runtime or the type when both are exported from the types file?